### DISCUSSION TITLE
怎么回复别人啊

### DISCUSSION THREAD

**Discussion by LuAn-afl** [share121/inter-knot#2903] [其他]
手机端怎么回复其他楼层啊？我只能回贴主的不能回其他楼层我看有些评论下面都有回复还有感谢Dave-12138大佬的米游社表情包脚本<img src="https://img-static.mihoyo.com/communityweb/upload/a2016a74dc5c31f7ee259e646415407c.png" alt="米游姬-抛心心" width="75" >

**Comment by nninng** [comment]
先点击写回复，然后往上滑，你就可以看到其他人的评论，然后下面就有一个可以打字的地方，点击就可以回复了

**Reply by nninng** [reply]
比如就像这样

**Reply by ChaoKaiADG** [reply]
ok啊谢谢大佬

**Reply by LuAn-afl** [reply]
已经会了感谢大佬

**Reply by QCJDBJ** [reply]
谢谢大佬我也会了

**Reply by qiuqiutata** [reply]
谢谢大佬

**Reply by Phaethonbell** [reply]
好使吗

**Reply by huangxingjun** [reply]
> 先点击写回复，然后往上滑，你就可以看到其他人的评论，然后下面就有一个可以打字的地方，点击就可以回复了

明白了，谢谢佬

**Reply by ll0930** [reply]
是这样吗

**Reply by lcshxl** [reply]
> > 先点击写回复，然后往上滑，你就可以看到其他人的评论，然后下面就有一个可以打字的地方，点击就可以回复了
> 
> >明白了，谢谢佬

你在看看你后面呢

**Reply by yekang40** [reply]
我不到啊

**Reply by jianci57** [reply]
> > > 先点击写回复，然后往上滑，你就可以看到其他人的评论，然后下面就有一个可以打字的地方，点击就可以回复了
> > 
> > 
> > > 明白了，谢谢佬
> 
> 你在看看你后面呢

你再看看你后面呢

**Comment by cyrilcc-z** [comment]
<img src="https://bbs-static.miyoushe.com/static/2024/07/31/fdc5e0e30e577fb208f29effa93510ed_8442612635207222552.png" alt="云璃 期待" width="75" >这样

**Reply by LuAn-afl** [reply]
已知晓

**Comment by jianjian114** [comment]
会写评论不会发帖子😭

**Reply by woxile233** [reply]
也是不会发帖子，只会回复。。

**Comment by HTZY-114** [comment]
666

**Comment by HTZY-114** [comment]
666

**Comment by lcshxl** [comment]
<img src="https://img-static.mihoyo.com/communityweb/upload/408018316100a8a8a846c68acedaa191.gif" alt="米游姬 卖萌" width="75" >
欧耶

**Comment by daoli668** [comment]
666

**Comment by Ace845186** [comment]
666

**Reply by yekang40** [reply]
qq

**Comment by girlsbandcry3** [comment]
zzz

**Comment by jianci57** [comment]
666

**Reply by jianci57** [reply]
666

**Reply by Meaowuu** [reply]
6

**Comment by kenlee26627** [comment]
666

**Comment by aWaSUbarBU** [comment]
666
